scripts/deploy-upgradable.js

```diff
@@ -1,13 +1,14 @@
 'use strict';
 require('dotenv').config();
+const _ = require('lodash/fp');
 const { Wallet, getDefaultProvider } = require('ethers');
 const { deployUpgradable, upgradeUpgradable, getProxy } = require('./upgradable');
 const readlineSync = require('readline-sync');
 const { outputJsonSync } = require('fs-extra');
 
 function getImplementationArgs(contractName, chain) {
-    if (contractName === 'AxelarGasService') return [chain.gasCollector];
-    if (contractName === 'AxelarDepositService') return [chain.gateway, chain.wrappedSymbol];
+    if (contractName === 'AxelarGasService') return [_.get('AxelarGasService.collector', chain)];
```

Comment:
does this return an error if the key doesn't exist? Should enforce that since it's easy to miss.

Reply:
Yes the ethers contract interface check for types, undefined would throw.
I am also adding a 0 check and revert to the service just in case